grid/recruitment/leads/maps/mock.suma

```diff
@@ -0,0 +1,17 @@
+profile = "recruitment/leads@1.0"
```

Comment:
For most ATSs lead is candidate with flag `sourced` or in state `LEAD`. Candidates and Leads shares the same ID in the ATSs, they represent the same entity.

From the point of proper harmonisation, handling leads as different entity has following pros:
- we are ready to harmonise ATSs which handles candidates and leads as separate entities
- creating lead with separate use case makes it easy to document the use cases, CreateLead and CreateCandidate do not share all the properties

Reply:
We should not project ATSs API directly into Comlinks. Instead we create use cases that the customer's app needs to use.

Reply:
My only concern here is that `recruitment/candidates` and `recruitment/leads` is too generic. Now we are designing the profiles for Job Board system. But the profile name doesn't reflect that.

Reply:
> The other option I see is to add the use case into recruitment/candidates profile.

Yeah, I think this is worth exploring because they have much in common as you mentioned, and we can differentiate between Leads and Candidates with use cases inside that profile.

For example we can have `ListLeads` which that client filters based on names or emails that got applied and use use case `CreateCandidate` to make that lead apply to a job.

Reply:
But since there will also be `CreateLead` for sourced leads, profile will be quite long and it might be confusing not to have it in different profiles. 🤷🏻‍♂️

Reply:
Another reason to have `CreateLead` in separate profile is that it is not supported by all providers.

Reply:
suggestion: Change profile name to `recruitment/create-lead`. Having single use case per profile is the best approach for ATS domain harmonisation. Look at the ATS domain providers comparison table and supported use cases across all of them.

Reply:
👍🏻

Reply:
shall we also move `CreateCandidate` to `recruitment/create-candidate`?

Reply:
> shall we also move CreateCandidate to recruitment/create-candidate?

Yes in separate PR.